dot/digest/README.md

```diff
@@ -0,0 +1,95 @@
+block and dispatches actions based on the messages the digests contain. The digest `Handler` supports digests from the
+BABE (authorship) and GRANDPA (finalisation) consensus engines - the following sections describe the messages in these
+digests and the actions Gossamer takes when it receives them.
+
+## BABE Messages
+
+[BABE](https://wiki.polkadot.network/docs/learn-consensus#block-production-babe) is a proof-of-stake block authorship
+algorithm that randomly assigns block authorship rights over periods of time referred to as "epochs". One of the
+benefits of BABE is that it is able to provide a blockchain runtime with secure, decentralized randomness. Furthermore,
+BABE seeks to ensure the liveness of a blockchain by defining multiple tiers of potential block producers for each
+epoch. BABE digests may contain 1 of 3 messages, each of which are described below.
+
+### Next Epoch
+
+This message is issued by the runtime on the first block of every epoch - it provides the BABE consensus engine with the
+authority set and randomness for the _next_ epoch.
+
+### Disabled
+
+A message of this type will contain the ID of an authority; this authority should cease all authority functionality and
+all other authorities should ignore any authority-related messages from the identified authority.
+
+### Next Config
+
+Messages of this type may only be issued in the first block of an epoch. These types of messages supply configuration
+parameters that should be applied from the _next_ epoch onwards. The parameters in this configuration relate to how
+backup authorities are selected.
+
+## GRANDPA Messages
+
+Deterministic [finality](https://wiki.polkadot.network/docs/glossary#finality) securely assures participants in a
+blockchain network that a block (and the
+[transactions](https://docs.substrate.io/v3/getting-started/glossary/#transaction) it contains) will not be reverted.
+Gossamer implements the [GRANDPA](https://wiki.polkadot.network/docs/learn-consensus#finality-gadget-grandpa) finality
+protocol, which, like BABE, assigns authority privileges to sets of nodes for given periods of time. The GRANDPA
+protocol defines the following consensus digest messages:
+
+### Scheduled Change
+
+These messages contain a list of new authority IDs and a delay, specified as a number of _finalised_ blocks, after which
+the change should be applied.
+
+### Forced Change
+
+This message is like that for scheduled changes, however the delay is calculated using _imported_ blocks (as opposed to
+finalised blocks), which means that the change is valid for multiple candidate chains.
+
+### Disabled
+
+A message of this type will contain the ID of an authority; this authority should cease all authority functionality and
+all other authorities should ignore any authority-related messages from the identified authority. _Note: Gossamer has
+not implemented supported for messages of this type._
```

Comment:
we don't support it for BABE either (it's essentially a no-op) because substrate hasn't implemented it either

Reply:
and by implemented, I mean that the digest exists in the chain, but substrate doesn't act upon it (so neither do we)